admin-dev/themes/new-theme/js/pages/logs/index.js

```diff
@@ -32,7 +32,7 @@ import SubmitGridActionExtension from '@components/grid/extension/submit-grid-ac
 import FiltersSubmitButtonEnablerExtension
   from '@components/grid/extension/filters-submit-button-enabler-extension';
 
-const $ = global.$;
+const {$} = global;
```

Comment:
Normal to use `global` here ?

Reply:
Don't ask me why they use global please :sob:

Reply:
This PR intend to help about code quality linked with a linter, Pierre shouldn't waste too moch time on refactoring and understanding everyone's code if the linter didn't return errors I think ^^